nncf/quantization/advanced_parameters.py

```diff
@@ -87,12 +101,25 @@ class QuantizationParameters:
     """
 
     num_bits: Optional[int] = None
-    mode: Optional[QuantizationMode] = None
+    scheme: Optional[QuantizationScheme] = None
```

Comment:
It's not possible, since it requires renaming for the dependent methods. For example, `HWConfig.get_quantization_mode_from_config_value` should be also renamed to return `scheme` instead of the `mode`.
Is it in the scope of this PR?

Reply:
I hope, yes.

Reply:
Also, the NNCF configuration files will become invalid when changing `mode -> scheme`.

Reply:
I've tried to do it in the whole NNCF. Not sure that pre-commit validation would be successful, but let's see.
Also, I think such a major change requires additional validation (at least e2e?) since it affects NNCF configuration files.
I would like to notify @vshampor about this huge change since it affects the whole NNCF.

Reply:
Reverted commit with the global renaming `mode -> scheme`.

Reply:
It looks like `nncf.qunatize(...)` will contain two `QuantizationMode` which mean different things.

Reply:
Renamed to `QuantizationScheme` and `QuantizationMode`. Updated imports for `QuantizationScheme` to keep backward compatibility.